api/v1alpha1/etcd_types_test.go

```diff
@@ -92,8 +92,8 @@ func getEtcd(name, namespace string) *Etcd {
 	deltaSnapshotPeriod := metav1.Duration{
 		Duration: 300 * time.Second,
 	}
-	imageEtcd := "eu.gcr.io/gardener-project/gardener/etcd:v3.4.13"
-	imageBR := "eu.gcr.io/gardener-project/gardener/etcdbrctl:v0.11.1"
+	imageEtcd := "eu.gcr.io/gardener-project/gardener/etcd:v3.4.13-bootstrap"
+	imageBR := "eu.gcr.io/gardener-project/gardener/etcdbrctl:v0.12.0-dev-2cb339762d03d5cc9c78bda855fdf48f309de9c3"
```

Comment:
```suggestion
	imageBR := "eu.gcr.io/gardener-project/gardener/etcdbrctl:v0.12.0"
```

Reply:
Did not want to change these yet, as `etcdbrctl:v0.12.0` hasn't been released yet, and testing in local gardener setup becomes cumbersome since the etcd resource deployed per control plane doesn't specify image versions, so the default image versions are used, and changing it becomes difficult as it gets reconciled unless we turn off gardenlet, hence made these image versions point to latest etcdbr version.
Once we make etcdbr release, all these image versions will be changed to reflect that. Hope that's ok.